Dockerfile

```diff
@@ -15,9 +15,6 @@
 # "slim" flavor because we don't need the big version.
 FROM node:10-buster-slim AS builder
 
-ENV NODE_ENV="production"
-ENV PORT=8000
-
```

Comment:
When I did this initially I wasn't sure of it. Now I think this was a mistake. Especially this disturbs the tests.